src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -58,10 +58,13 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
                 assemblyLocation = cashedAssemblyLocation;
             }
 
-            var assemblyPath = Path.GetFullPath(assemblyLocation);
```

Comment:
Problem was here.

From MSDN:
The file or directory specified by path is not required to exist. For example, if `c:\temp\newdir` is the current directory, calling **GetFullPath** on a file name such as `test.txt` returns `c:\temp\newdir\test.txt`. The file need not exist.

So, we have file, that doesn't exist in fact. They are BuiltIn.dll and Operators.dll.  In DynamoStandalone `Path.GetFullPath` will return `bin` folder. But in Dynamo on Revit the same code will return Revit folder. And that will be incorrect.

My intent was, if file is not found => use base Dynamo  folder to search for needed .xml file.